webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    bundle: path.resolve(__dirname, './src/index.js'),
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: '[name].js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      title: 'to-do-list',
+      filename: 'index.html',
+      template: './src/index.html',
+    }),
+  ],
+
+};
```

Comment:
- [ ] Nice work using Webpack for the implementation of the project :+1:  However Webpack **dev server** is not set up in your Webpack configuration. Kindly make sure Webpack **dev server** is set up and running with no errors